src/Controls/samples/Controls.Sample.SingleProject/Maui.Controls.Sample.SingleProject.csproj

```diff
@@ -40,7 +40,7 @@
   </ItemGroup>
   <ItemGroup>
     <MauiImage Include="Resources\AppIcons\appicon.svg" ForegroundFile="Resources\AppIcons\appicon_foreground.svg" IsAppIcon="true" />
-    <MauiAsset Include="Resources\Raw\*" />
+    <MauiAsset Include="Resources\Raw\*" Link="%(Filename)%(Extension)" />
```

Comment:
I used `%(Link)` here which isn't ideal, because iOS doesn't allow a `@(Content)` item in a `Resources` folder:
```
Resources/Raw/index.html The name 'Resources' is reserved and cannot be used.
```
Then in the `<WebView/>`, we just use `index.html` directly.

I think this would work fine for Blazor and the `wwwroot` directory. They should need to use `%(Link)` like this.

If you had something like:
```
<MauiAsset Include="wwwroot\**\*" />
```

On iOS you'll get a file you can open such as `File.OpenRead("wwwroot/index.html")`.

On Android, you'll get `file:///android_asset/wwwroot/index.html`. Or you can use the `AssetManager` to get a `Stream` for `wwwroot/index.html`.